qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -0,0 +1,39 @@
+- balloon_memhp:
+    type = balloon_memhp
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 64G
+    login_timeout = 600
+    no Host_RHEL.6
+    no RHEL.5
+    only Linux Win2008.x86_64 Win2012.x86_64 Win2016.x86_64 Win2019.x86_64
```

Comment:
Prefer `no ***`, otherwise, we need update again if new windows guest come out.

Reply:
when use `no ***`,  very old windows guest can be ignore from my point of view.

Reply:
@lijinlijin What's your opinion regarding this? Besides i386, win7, win8, win10, what else do you have in mind to exclude?

Reply:
win8.1 also need to be excluded.
"only xxx" and "no xxx" actually is same as memory cpu hotplug is supported on servers.
If new windows desktop come out, we still need to add no new-desktop to the list.

Reply:
@lijinlijin Thanks. Then I will keep the current 'only'.

Reply:
> win8.1 also need to be excluded.
> "only xxx" and "no xxx" actually is same as memory cpu hotplug is supported on servers.
> If new windows desktop come out, we still need to add no new-desktop to the list.

got your point, then it is ok for, thanks !